ccd-definition/Jurisdiction.json

```diff
@@ -2,7 +2,7 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "PUBLICLAW",
```

Comment:
I guess that ID can remain unchanged. New case type will just appear under jurisdiction that is in use. I guess having rename only is sensible.